Assets/Tests/InputSystem/Plugins/UITests.cs

```diff
@@ -2886,4 +3065,122 @@ protected override void OnApplicationFocus(bool hasFocus)
             base.OnApplicationFocus(hasFocus);
         }
     }
+
+    private static KeyValuePair<string, object> OneEvent(string property, object value)
```

Comment:
The previous way to check event sequences became extremely tedious to maintain so I spent some time trying to come up with a better system. Basically, instead of manually checking specific entries in the event array, you write

```
Assert.That(myReceiver.events,
    EventSequence(
        // Ensure all events in the sequence have the same value for one property.
        AllEvents("propertyOnTheEvent", value),

        // Ensure a specific sequence of values for a single property. E.g. here
        // the sequence must contain two events, one with the first type and
        // one with the second type.
        OneEvent("type", EventType.PointerDown),
        OneEvent("type", EventType.PointerMove)
```